modules/UI/UI.js

```diff
@@ -396,7 +396,8 @@ UI.unbindEvents = () => {
 UI.addLocalStream = track => {
     switch (track.getType()) {
     case 'audio':
-        VideoLayout.changeLocalAudio(track);
+        // Local audio is not rendered so no further action is needed at this
+        // point.
```

Comment:
From what I remember it never gets attached. Nothing accesses local audio because local audio.